cpp/code/protos/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+# Based on https://www.f-ax.de/dev/2020/11/08/grpc-plugin-cmake-support.html
```

Comment:
Hmm, is there a licensing concern here…?

We could also base this on how Flight generates its protos. We can just include the Proto sources directly in our binary instead of as a library.

Reply:
Oh and the repo with the example code has no license. Which sounds like it means there is def an license issue.

https://github.com/faaxm/exmpl-cmake-grpc